docs/source/intro/datasets.rst

```diff
@@ -36,6 +36,46 @@ formats:
    IEEE Xplore, Scopus and ScienceDirect. Citation managers Mendeley, RefWorks,
    Zotero, and EndNote support the RIS file format as well.
 
+For parsing RIS file format, the software uses a Python RIS files parser and reader (`rispy
+    <https://pypi.org/project/rispy/>`_). Successful import/export depends on a proper
+    data set structure. To validate your data set, the complete default mapping can be found
+    on the developer's `GitHub page <https://github.com/MrTango/rispy#complete-default-mapping>`_.
```

Comment:
Please resolve warnings

Reply:
The documentation looks to be riddled with these warnings. This might be due to having an outdated version.